data-catalog/overlays/dev/argocd-rb.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: authorization.openshift.io/v1
+kind: RoleBinding
+metadata:
+  name: argocd-manager-rolebinding
+  namespace: dh-dev-analytics-factory
```

Comment:
I think the namespace should be specified in the kustomize file for all of these, not on the objects.